tic-tac-toe/src/scripts/calculate-next-move.js

```diff
@@ -0,0 +1,51 @@
+const firsPlayerRatingFactor = 10;
+const secondPlayerRatingFactor = 1;
+const twoItemsCombinationCost = 100;
+const oneItemsCombinationCost = 10;
+const angleOffset = 1;
```

Comment:
it's better to add an explanation here. why we use it, why we need it

Reply:
I deleted these unnecessary variables. They are left from the previous version of the algorithm.